Themis/Views/Assessment/AssessmentSubmissionLoaderView.swift

```diff
@@ -9,24 +9,17 @@ import SwiftUI
 
 struct AssessmentSubmissionLoaderView: View {
     @StateObject var avm = AssessmentViewModel(readOnly: false)
-    @StateObject var cvm = CodeEditorViewModel()
 
     var submissionID: Int
     let exercise: Exercise
 
     var body: some View {
         AssessmentView(
             vm: avm,
-            cvm: cvm,
             ar: avm.assessmentResult,
-            exercise: exercise
+            exercise: exercise,
+            submissionId: submissionID
         )
```

Comment:
Like this, the `AssessmentSubmissionLoaderView` is entirely empty and serves no purpose. Why was the loading of the assessment moved away from it?

Reply:
As mentioned above, so that cvm can be a state object in assessment view